workflow-examples/src/main/java/com/redhat/parodos/examples/prebuilt/migrationtoolkit/ProcessAnalysisTask.java

```diff
@@ -83,7 +82,10 @@ public WorkReport execute(WorkContext workContext) {
 			MTAAnalysisReport.AnalysisIncidents incidents = MTAAnalysisReport.extractIncidents(issueSummaries);
 
 			if (incidentsPredicate.test(incidents)) {
-				sendNotification();
+				notificationSender.send(
+						passOption.getDisplayName(),
+						"The MTA report summary passes the specified criteria in the workflow. { %s }"
+								.formatted(incidentsPredicate) + passOption.getDescription());
```

Comment:
can you add the last argument which is concatenated by + to the formatted string?